akka-stream-tests-tck/src/test/scala/akka/stream/tck/AkkaIdentityProcessorVerification.scala

```diff
@@ -18,7 +18,7 @@ abstract class AkkaIdentityProcessorVerification[T](env: TestEnvironment, publis
   with TestNGSuiteLike with ActorSystemLifecycle {
 
   def this(printlnDebug: Boolean) =
-    this(new TestEnvironment(Timeouts.defaultTimeoutMillis, printlnDebug), Timeouts.publisherShutdownTimeoutMillis)
+    this(new TestEnvironment(Timeouts.defaultTimeoutMillis, Timeouts.defaultTimeoutMillis, printlnDebug), Timeouts.publisherShutdownTimeoutMillis)
```

Comment:
the point of the ticket was to reduce this timeout (it's like `expectNoMsg` right?) and that should dramatically reduce the total time it takes to run the tests